puppet/zulip/manifests/app_frontend_once.pp

```diff
@@ -60,4 +60,12 @@
     minute => '22',
   }
 
+  # 1st day every 3 months.
+  zulip::cron { 'update-stream-active-status':
```

Comment:
Do we want this to run in a specific timezone too?

Reply:
I ... don't know what you mean by that?  Cron jobs all run in time specified as UTC.